string map_name
---
bool status
